kernelci/api/helper.py

```diff
@@ -132,11 +132,14 @@ def create_job_node(self, job_config, input_node,
             'name': job_config.name,
             'path': input_node['path'] + [job_config.name],
             'group': job_config.name,
-            'artifacts': input_node['artifacts'],
+            'artifacts': {},
             'data': {
                 'kernel_revision': input_node['data']['kernel_revision'],
             },
         }
+        # Specific artifacts we want to keep from the parent node
```

Comment:
Maybe, but why? Is it always needed? Or just for specific jobs? A more detailed comment would be really helpful when we'll try to reduce the tech debt.

Reply:
Thanks, the more detailed comment is already a good start. This `python.jinja2` template feels wrong in many ways tbh, I'll try to improve it a bit.

Reply:
Tried to address this with https://github.com/kernelci/kernelci-core/pull/2428 so you can hopefully avoid single-casing this artifact ;)

Reply:
> I feel like we should address the reimplementation of the job generation code some day when we find the time to do it.

Agreed, sounds like a good task for post-M1 tech debt reduction ;)